app/Controllers/feedController.php

```diff
@@ -385,10 +386,15 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 				if ($simplePiePush) {
 					$simplePie = $simplePiePush;	//Used by WebSub
 				} elseif ($feed->kind() === FreshRSS_Feed::KIND_HTML_XPATH) {
-					$simplePie = $feed->loadHtmlXpath(false, $isNewFeed);
+					$simplePie = $feed->loadHtmlXpath();
 					if ($simplePie == null) {
 						throw new FreshRSS_Feed_Exception('HTML+XPath Web scraping failed for [' . $feed->url(false) . ']');
 					}
+				} elseif ($feed->kind() === FreshRSS_Feed::KIND_XML_XPATH) {
+					$simplePie = $feed->loadHtmlXpath();
+					if ($simplePie == null) {
```

Comment:
suggestion :  use strict comparaison
```suggestion
					if ($simplePie === null) {
```

Reply:
Done